IBPSA/Resources/C-Sources/WeeklySchedule.h

```diff
@@ -0,0 +1,67 @@
+/*
+
+  This code implements a weekly schedule. Input text example:
+
+#some comment
+double  tab1(3,5) # another comment
+# blabla
+mon:0:0:10              -       3       1       -
+tue,thu:20:30:59        123     -       45      -
+wed,sat,sun             12      1       4       -
+
+
+	License: BSD3
+
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+ Changelog:
```

Comment:
We usually don't have changelog for C code.

Reply:
The MSL CombiTimeTable has it and I figured it's useful?

Reply:
Also, I'd like to be recognised as a contributor/author in case some copies the code into a separate library.

Reply:
We can add the author if you like, but I would not add a license as this is already handled at the top level. Otherwise we open up the situation that others also like to add licenses, which may not be identical with the top level license (as is the case here) which opens up legal complexity that I rather don't want to deal with if we can avoid it as we did in the past.

Reply:
Okay!

Reply:
I removed the license.